src/fileseq/filesequence.py

```diff
@@ -822,6 +1021,20 @@ def getPaddingChars(num):
         else:
             return "@" * num
 
+    @staticmethod
+    def getUsdPaddingChars(num):
+        """
+        Given a particular amount of padding, return the proper padding
+        characters in USD format.
```

Comment:
The usdPadding stuff was a last minute addition, and I don't really like it, but I wanted to put something in there so we think about the problem. Keeping this consistent with the Go and C++ versions would be ideal. The main reason I didn't make a USD subclass or similar up front is that it isn't always easy to choose the correct handling when creating a FileSequence from a string, maybe found in a large database of such things. If we specify that for external storage all FileSequences should use DEFAULT padding it might work?

Reply:
Ya I agree that there isn't a logical way to decide what type of padding to use simply from the string, since its entirely application specific. And we happened to have gone with the hash=4 approach. I was wrong in mentioning printf and houdini since those are pad *parsing* formats and not a definition for how to expand the sequence pattern to a file name. Its really just about hash=4 (default) vs hash=1 (usd) which is what the C++ and Go versions implement with constants. 
I'm glad to hear you say you weren't entirely happy with the "usd" wording, since I usually try not to bake in any product names right into the API surface which could end up deprecated and as cruft. Constants/enums don't feel that bad to me since you can always just stop handling the values but they dont hang off the classes as public details.
I guess a quick alternative would be to subclass a `UsdFileSequence` that swaps out the `PAD_MAP`.